Formula/php.rb

```diff
@@ -152,22 +162,29 @@ def install
       --with-unixODBC=#{Formula["unixodbc"].opt_prefix}
       --with-webp-dir=#{Formula["webp"].opt_prefix}
       --with-xmlrpc
-      --with-xsl
-      --with-zlib
+      --with-xsl#{include_path}
+      --with-zlib#{include_path}
     ]
 
     if MacOS.version < :lion
       args << "--with-curl=#{Formula["curl"].opt_prefix}"
     else
-      args << "--with-curl"
+      args << "--with-curl#{include_path}"
     end
 
     if build.devel?
       args << "--with-ldap=#{Formula["openldap"].opt_prefix}"
     else
-      args << "--with-ldap"
+      args << "--with-ldap#{include_path}"
     end
 
+    if MacOS.version == :mojave
+      args << "--with-iconv=#{Formula["libiconv"].opt_prefix}"
+    end
+
+    # buildconf required due to linking bug patch
+    system "./buildconf", "--force"
```

Comment:
This needs to be above the `inreplace "configure"` lines otherwise it will wipe them out.

Reply:
Will fix, thanks for the pointer.